tests/02_00_06-2wp-new-minimum.js

```diff
@@ -1,117 +1,130 @@
+        expect(senderAddressBalanceFinal).to.equal(0);
 
-        expect(Number(initialSenderBalance)).to.equal(0);
-        expect(Number(finalSenderBalance)).to.equal(0);
-        expect(Number(initialDerivedAddressBalance)).to.equal(0);
-        expect(Number(finalDerivedAddressBalance)).to.equal(rsk.btcToWeis(AMOUNT_TO_LOCK_IN_BTC));
+        const recipientRskAddressBalanceFinal = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+        expect(recipientRskAddressBalanceFinal).to.equal(btcEthUnitConverter.btcToWeis(MINIMUM_PEGIN_VALUE_IN_BTC));
     });
 
-    it('should not peg-in when sending below minimum value', async () => {
-        const AMOUNT_TO_LOCK_IN_BTC = btcClient.nodeClient.satoshisToBtc(minimumPeginValueInSatoshis - 1);
+    it('should not peg-in and not refund when sending below minimum value', async () => {
+        const PEGIN_VALUE_IN_BTC = MINIMUM_PEGIN_VALUE_IN_BTC - 0.001
 
         // Create legacy type address to use as sender
-        let senderAddressInformation = await btcClient.generateBtcAddress('legacy');
-        let initialSenderBalance = await btcClient.getAddressBalance(senderAddressInformation.address);
+        const senderAddressInfo = await btcTxHelper.generateBtcAddress('legacy');
 
-        // Get the RSK address where the funds should be locked to
-        let rskDerivedAddress = keyUtils.getRskAddress(senderAddressInformation.privateKey);
-        let initialDerivedAddressBalance = await rskClient.eth.getBalance(rskDerivedAddress);
+        await btcTxHelper.fundAddress(senderAddressInfo.address, PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
 
-        await btcClient.fundAddress(senderAddressInformation.address, AMOUNT_TO_LOCK_IN_BTC + btcClient.getFee());
+        //get federation address
+        const federationAddress = await bridge.methods.getFederationAddress().call();
+        const federationAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(federationAddress));
 
-        // Execute peg-in
-        await _2wpUtilsLegacy.sendPegin(rskClient, btcClient, senderAddressInformation, AMOUNT_TO_LOCK_IN_BTC);
-        const rskClients = Runners.hosts.federates.map(federate => rsk.getClient(federate.host));
-        await rskUtilsLegacy.triggerPegoutEvent(rskClients);
+        // Execute peg-in (pegin is not registered in the bridge -> Someone sent to the federation UTXOs amount less than 50000000 satoshis)
+        await sendPegin(rskTxHelper, btcTxHelper, senderAddressInfo, PEGIN_VALUE_IN_BTC);
 
-        // Assert
-        let finalSenderBalance = await btcClient.getAddressBalance(senderAddressInformation.address);
-        let finalDerivedAddressBalance = await rskClient.eth.getBalance(rskDerivedAddress);
+        const federationAddressBalanceAfterPegin = Number(await btcTxHelper.getAddressBalance(federationAddress));
+        expect(federationAddressBalanceAfterPegin).to.be.equal(Number(federationAddressBalanceInitial + PEGIN_VALUE_IN_BTC));
```

Comment:
The Fed Balance Should Remain The Same, Since The Pegin Wasn't Successful

Reply:
i think in this case the user is losing the funds. am i right @marcos-iov ?

Reply:
The Funds Would Be Lost, However, The Federation Balance Would Increase On The BTC Network, While The UXTO Wont Be Registered In The Bridge - We Should Assert This